bundle-size/github-utils.js

```diff
@@ -129,50 +127,27 @@ class GitHubUtils {
     ).includes(username);
   }
 
-  /**
-   * Convert a team slug to a team id.
-   *
-   * @param {string} teamName the team slug (`organization/team`).
-   * @return {number} the team id.
-   */
-  async getTeamId_(teamName) {
-    const cacheKey = `${CACHE_TEAM_ID_KEY}/${teamName}`;
-    const [org, teamSlug] = teamName.split('/');
-    let teamId = this.cache.get(cacheKey);
-    if (!teamId) {
-      this.log(`Cache miss for ${cacheKey}. Fetching from GitHub...`);
-      teamId = await this.github.teams
-        .getByName({org, team_slug: teamSlug})
-        .then(result => result.data.id);
-      this.log(
-        `Fetched team id ${teamId} for team ${teamName} from GitHub. Caching ` +
-          `for ${CACHE_TEAM_ID_TTL_SECONDS} seconds.`
-      );
-      this.cache.set(cacheKey, teamId, CACHE_TEAM_ID_TTL_SECONDS);
-    }
-    return teamId;
-  }
-
   /**
    * Get a list of all unique team members by team names.
    *
-   * @param {!Array<string>} teamNames names of full team slug namesto get
-   *   members of.
+   * @param {!Array<string>} teamNames names of full team slug names
+   *   ("org/team_slug") format to get members of.
    * @return {!Array<string>} all members of the provided teams.
    */
   async getTeamMembers(teamNames) {
     const allTeamMembersPromises = teamNames.map(async teamName => {
-      const teamId = await this.getTeamId_(teamName);
-      const cacheKey = `${CACHE_TEAM_MEMBERS_KEY}/${teamId}`;
+      const cacheKey = `${CACHE_TEAM_MEMBERS_KEY}/${teamName}`;
       let teamMembers = this.cache.get(cacheKey);
       if (!teamMembers) {
         this.log(`Cache miss for ${cacheKey}. Fetching from GitHub...`);
+        const [org, teamSlug] = teamName.split('/', 2);
         teamMembers = await this.github.teams
-          .listMembers({team_id: teamId})
+          .listMembersInOrg({org, team_slug: teamSlug})
           .then(result => result.data.map(user => user.login));
         this.log(
-          `Fetched team members [${teamMembers.join(', ')}] for team id ` +
-            `${teamId}. Caching for ${CACHE_TEAM_MEMBERS_TTL_SECONDS} seconds.`
+          `Fetched team members [${teamMembers.join(', ')}] for team ` +
+            `${teamName}. Caching for ${CACHE_TEAM_MEMBERS_TTL_SECONDS} ` +
+            'seconds.'
```

Comment:
Note to nobody: This could be shared infra, owners does its own cache-refresh pattern with teams too

Reply:
Ack